### PR TITLE
First Video Lecture - (Intro to Javascript)

### PR DESCRIPTION
**Pull request by isuyashpatel**

### CODE REVIEW

File: pages/docs/cohort-1/week-1/javascript.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+## Key Points
+
+- Computer Understand 0's and 1's
+- [High-level and Low-level language](https://www.google.com/search?q=low+level+language+and+high+level+language)
+- JavaScript is High-level programming language
+- [ECMAScript (or ES)](https://en.wikipedia.org/wiki/ECMAScript) is the standardized scripting language specification upon which JavaScript is based. JavaScript engines are the software components responsible for interpreting and executing JavaScript code. Various browsers and environments have their own JavaScript engines that implement the ECMAScript specifications. Some of the well-known JavaScript engines include: [v8](https://nodejs.org/en/learn/getting-started/the-v8-javascript-engine), [SpiderMonkey](https://spidermonkey.dev), [JavaScriptCore](https://developer.apple.com/documentation/javascriptcore),etc.
+
+## Data Types
+
+There are two types of data [Primitive and Non-Primitive](https://letmegooglethat.com/?q=primitive+and+non-primitive+data+types).
```
Comment (thatbeautifuldream) [inline pages/docs/cohort-1/week-1/javascript.mdx:16]: lets replace the letmegooglethat links with actual search queries for non-intrusive behaviour.

https://www.google.com/search?q=Primitive+and+Non-Primitive

File: pages/docs/cohort-1/week-1/javascript.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+Some Additional questions
+1- Write a Function which return sum between 1 to 100.
+2- Fibonacci services
+3- Stary Pattern 
+*/
+```
+
+## Core Concept
+
+- [Single-threaded and Multi-threaded](https://letmegooglethat.com/?q=Single-threaded+and+Multi-threaded)
```
Comment (thatbeautifuldream) [inline pages/docs/cohort-1/week-1/javascript.mdx:88]: Please replace the link with : 

https://www.google.com/search?q=Single-threaded+and+Multi-threaded

File: pages/docs/cohort-1/week-1/meta.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "basic-javascript": "Intro to Javascript"
```
Comment (thatbeautifuldream) [inline pages/docs/cohort-1/week-1/meta.json:2]: Lets keep the titles and the url slug simmilar like if its Intro to Js lets keep the slug :

`intro-to-javasctipt`

**Review by thatbeautifuldream — CHANGES_REQUESTED** [review]

**Comment by isuyashpatel** [comment]
@thatbeautifuldream fixed

**Review by thatbeautifuldream — APPROVED** [review]
LGTM

**Pull request merged**